src/createAnimatedComponent.js

```diff
@@ -344,6 +355,12 @@ export default function createAnimatedComponent(Component) {
       return props;
     }
 
+    _obtainViewName() {
```

Comment:
obtain is a bit weird wording, it means get+hold whereas the method does simply get the name and returns it

Reply:
fixed 6609d065d654dd05e644bee42dfdb62405907c7e